microsoft365defender_consts.py

```diff
@@ -0,0 +1,156 @@
+# File: microsoft365defender_consts.py
+#
+# Copyright (c) 2022 Splunk Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied. See the License for the specific language governing permissions
+# and limitations under the License.
+
+# Define your constants here
+DEFENDER_USER_AGENT = "M365dPartner-Splunk-SOAR/{product_version}"
+DEFENDER_AUTHORIZATION_HEADER = "Bearer {token}"
+
+DEFENDER_SOAR_BASE_URL = '{soar_base_url}rest'
+DEFENDER_SOAR_SYS_INFO_URL = '/system_info'
+DEFENDER_SOAR_ASSET_INFO_URL = '/asset/{asset_id}'
+DEFENDER_TC_FILE = 'oauth_task.out'
+DEFENDER_CONFIG_TENANT_ID = 'tenant_id'
+DEFENDER_CONFIG_CLIENT_ID = 'client_id'
+DEFENDER_CONFIG_CLIENT_SECRET = 'client_secret'  # pragma: allowlist secret
+
+DEFENDER_TOKEN_STRING = 'token'
+DEFENDER_ACCESS_TOKEN_STRING = 'access_token'
+DEFENDER_CODE_STRING = 'code'
+DEFENDER_REFRESH_TOKEN_STRING = 'refresh_token'
+DEFENDER_ID_TOKEN_STRING = 'id_token'
+DEFENDER_CLIENT_CREDENTIALS_STRING = 'client_credentials'
+DEFENDER_BASE_URL_NOT_FOUND_MSG = 'Phantom Base URL not found in System Settings. ' \
+                                     'Please specify this value in System Settings'
+DEFENDER_AUTHORIZE_URL = '/{tenant_id}/oauth2/authorize?client_id={client_id}&redirect_uri={redirect_uri}' \
+                            '&response_type={response_type}&state={state}&resource={resource}'
+DEFENDER_RECEIVED_ALERT_INFO_MSG = 'Received alert info'
+DEFENDER_HTTP_401_STATUS_CODE = '401'
+DEFENDER_UNAUTHORIZED_CLIENT_ERROR_MSG = 'unauthorized_client'
+DEFENDER_INVALID_TENANT_ID_FORMAT_ERROR_CODE = 'AADSTS900023'
+DEFENDER_INVALID_TENANT_ID_NOT_FOUND_ERROR_CODE = 'AADSTS90002'
+DEFENDER_ALERTS_ENDPOINT = '/security/alerts_v2'
+DEFENDER_RUN_QUERY_ENDPOINT = '/security/runHuntingQuery'
+DEFENDER_SERVER_TOKEN_URL = '/{tenant_id}/oauth2/token'
+DEFENDER_LOGIN_BASE_URL = 'https://login.microsoftonline.com'
+DEFENDER_RESOURCE_URL = 'https://graph.microsoft.com'
+DEFENDER_MSGRAPH_API_BASE_URL = 'https://graph.microsoft.com/beta'
```

Comment:
@spopat-crest 
Please create a backlog ticket to update API once MS releases graph API

Reply:
Created the ticket: https://splunk.atlassian.net/browse/PAPP-27138